ClientLib/src/main/java/com/o3dr/services/android/lib/drone/connection/ConnectionParameter.java

```diff
@@ -277,7 +277,7 @@ public static ConnectionParameter newSoloConnection(String ssid, @Nullable Strin
             eventsDispatchingPeriod);
     }
 
-    private ConnectionParameter(@ConnectionType.Type int connectionType, Bundle paramsBundle){
+    public ConnectionParameter(@ConnectionType.Type int connectionType, Bundle paramsBundle){
```

Comment:
As far as I remember class _BasicTest_ complains about the access to this constructor while compilation:
https://github.com/dronekit/dronekit-android/blob/develop/ClientLib/src/androidTest/java/org/droidplanner/services/android/impl/BasicTest.java#L60